modules/accessing-the-administration-interface-for-basic-workbenches.adoc

```diff
@@ -0,0 +1,29 @@
+:_module-type: PROCEDURE
+
+[id='accessing-the-administration-interface-for-basic-workbenches_{context}']
+= Accessing the administration interface for basic workbenches
+
+[role='_abstract']
+You can use the administration interface to control basic workbenches in your {productname-long} environment.
+
+.Prerequisite
+
+* You have logged in to {productname-short} as a user with {productname-short} administrator privileges. 
+
+.Procedure
+** To access the administration interface for basic workbenches from {productname-short}, perform the following actions:
+... In {productname-short}, in the *Applications* section of the left menu, click *Enabled*.
+... Locate the *Start basic workbench* tile and click *Launch application*.
+... On the page that opens when you launch a basic workbench, click the *Administration* tab.
++
+The *Administration* page opens.
+
+** To access the administration interface for basic workbenches from JupyterLab, perform the following actions:
+... Click *File* -> *Hub Control Panel*.
```

Comment:
That's nonsense, there is nothing that reads _Hub Control Panel_ in the UI. I'll doublecheck later. Looks like leftover from the JupyterHub days.

Reply:
Hmm, actually, ... yeah, gotta check

Reply:
@coderabbitai do you think users of OpenShift AI can Click *File* -> *Hub Control Panel*?

Reply:
The step is prefixed with "To access the administration interface for basic workbenches ****"